packages/babel-helper-evaluate-path/src/index.js

```diff
@@ -109,9 +109,14 @@ function evaluateIdentifier(path) {
 // if the binding is in program scope
 // all it's references inside a different function should be deopted
 function shouldDeoptBasedOnScope(binding, refPath) {
+  if (binding.kind === "const") {
+    return false;
+  }
+
```

Comment:
why only `const` ?. why does this repeat twice - one below?

Reply:
because let variable values can change ?